libs/shared/src/consts/providers/provider.enum.ts

```diff
@@ -107,6 +107,7 @@ export enum ChatProviderIdEnum {
   Zulip = 'zulip',
   GrafanaOnCall = 'grafana-on-call',
   GetStream = 'getstream',
+  RocketChat = 'rocket-chat',
```

Comment:
it's fine but there is mismatch between square logo variant and provider's id (`rockerchat` and `rocket-chat`).

You should rename square logo file name to match provider's id, so it would be `rocket-chat.svg` instead `rocketchat.svg`.

Reply:
renamed